rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityFilter.vue

```diff
@@ -216,9 +218,14 @@
       </div>
       <template #footer>
         <btn @click="filterOpen = false">{{ $t("cancel") }}</btn>
-        <btn type="primary" class="btn btn-primary" @click="search">{{
-          $t("search")
-        }}</btn>
+        <btn
+          ref="searchButton"
+          type="primary"
+          class="btn btn-primary"
+          data-testid="searchfilter"
+          @click="search"
+          >{{ $t("search") }}</btn
+        >
```

Comment:
Please add a new line here so that the formatting will be:

```suggestion
          >
            {{ $t("search") }}
          </btn>
```